app/models/request/SaveExpense.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.request
+
+import models.responses.UkOtherPropertyExpenses
+import play.api.libs.json.{Json, OFormat}
+
+final case class SaveExpense(ukOtherPropertyExpenses: UkOtherPropertyExpenses)
```

Comment:
Let's avoid having a named request body and also create our own model.

Reply:
Will do this at a later PR. Ideally I would like to get this over the line now